electron/js/menu/system.js

```diff
@@ -226,6 +233,15 @@ var toggleFullScreenTemplate = {
   },
 };
 
+var toggleAutoLaunchTemplate = {
+  i18n: 'menuAutoStart',
+  type: 'checkbox',
+  checked: init.restore('shouldAutoLaunch'),
```

Comment:
As we discussed in one other PR.. the `init.restore` is always used with the default value (you never know what can go wrong with the filesystem) so I would say use `false` wherever it is used..